src/excel/Workbook.js

```diff
@@ -38,6 +38,24 @@ export class Workbook extends NamedItemContainer {
     return new Worksheet(this._oneDrive, `${this._uri}/worksheets`, name, this._log);
   }
 
+  async createWorksheet(sheetName) {
+    const uri = `${this.uri}/worksheets`;
+    await this._oneDrive.doFetch(uri, false, {
+      method: 'POST',
+      body: { name: sheetName },
+      headers: { 'content-type': 'application/json' },
+    });
+    return this.worksheet(sheetName);
+  }
+
+  async deleteWorksheet(sheetName) {
+    const uri = `${this.uri}/worksheets/${sheetName}`;
+    return this._oneDrive.doFetch(uri, false, {
```

Comment:
the return type doesn't match the one in the type (`Promise<void>`). 
```suggestion
    await this._oneDrive.doFetch(uri, false, {
```